src/Integration/Notifications/NotificationIndicatorViewModel.cs

```diff
@@ -60,6 +70,14 @@ internal NotificationIndicatorViewModel(Action<Action> uiThreadInvoker, ITimer a
 
             autocloseTimer.Elapsed += OnAutocloseTimerElapsed;
             ClearUnreadEventsCommand = new RelayCommand(() => HasUnreadEvents = false);
+
+            NavigateToNotification = new DelegateCommand(parameter =>
+            {
+                var notification = (SonarQubeNotification) parameter;
+                vsBrowserService.Navigate(notification.Link.ToString());
+                IsToolTipVisible = false;
```

Comment:
`IsToolTipVisible = false;` was part of the previous code-behind: https://github.com/SonarSource/sonarlint-visualstudio/blob/6a9c7fd05e657bb2a926413598089b3156fa4504/src/Integration/Notifications/NotificationIndicator.xaml.cs#L43